src/Math/Matrix4.elm

```diff
@@ -45,6 +45,11 @@ transform = Native.MJS.v3mul4x4
 identity : Mat4
 identity = Native.MJS.m4x4identity
 
+{-| Computes the inverse of the given matrix m.
+-}
+inverse : Mat4 -> Mat4
```

Comment:
We'll definitely need to figure that out since calling `throw` in JavaScript is not acceptable.